docfx/yaml_emit.cc

```diff
@@ -0,0 +1,23 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "docfx/yaml_emit.h"
+
+namespace docfx {
+
+YAML::Emitter& operator<<(YAML::Emitter& e, std::string_view rhs) {
```

Comment:
Observation: It is a little strange that this wouldn't be found via ADL, but I'm not sure what the alternative is.

Reply:
Yup.